runner/src/main/openapi/runner.yaml

```diff
@@ -674,6 +675,139 @@ components:
           x-field-extra-annotation: "@com.redis.om.spring.annotations.Indexed"
           allOf:
             - $ref: '#/components/schemas/RunnerSecurity'
+      required:
+        - id
+        - name
+        - ownerId
+        - solutionId
+        - runTemplateId
+        - organizationId
+        - workspaceId
+        - creationDate
+        - lastUpdate
+        - ownerName
+        - datasetList
+        - parametersValues
+        - validationStatus
+        - security
+
+    # CRUD Operation Schemas
+    RunnerCreateRequest:
+      type: object
+      description: Request object for creating a new runner
+      properties:
+        name:
+          type: string
+          description: the Runner name
+          minLength: 1
+        description:
+          type: string
+          description: the Runner description
+        tags:
+          type: array
+          description: the list of tags
+          items:
+            type: string
+        solutionId:
+          type: string
+          readOnly: true
+          description: the Solution Id associated with this Runner
+          minLength: 1
+          pattern: '^sol-\w{10,20}'
+          example: "sol-123456aBcDeF"
+        parentId:
+          type: string
+          description: the Runner parent id
+          minLength: 1
```

Comment:
we can even specified pattern for`parentId`, like `'^r-\w{10,20}'`  and remove the minLength